webpack.config.js

```diff
@@ -156,6 +164,6 @@ if ( config.isEnabled( 'webpack/persistent-caching' ) ) {
 	webpackConfig.plugins.unshift( new HardSourceWebpackPlugin( { cacheDirectory: path.join( __dirname, '.webpack-cache', 'client' ) } ) );
 }
 
-webpackConfig.module.loaders = [ jsLoader ].concat( webpackConfig.module.loaders );
+webpackConfig.module.postLoaders = [ jsLoader ];
```

Comment:
>Having the babel transformation happen in a postLoader worries me a little. Was this just for the `import` syntax change?

Yes, if I recall correctly, it was so that:

(a) The async component loader could return ES6
(b) The Babel plugin would be run last to strip the `async-component!` import prefix